Source/Blazorise/Components/Link/Link.razor.cs

```diff
@@ -24,9 +24,20 @@ protected override void BuildClasses( ClassBuilder builder )
     {
         builder.Append( ClassProvider.Link() );
         builder.Append( ClassProvider.LinkActive( Active ) );
+        builder.Append( ClassProvider.LinkUnstyled( Unstyled ) );
 
         base.BuildClasses( builder );
     }
 
     #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// Remove styles from Link component.
+    /// </summary>
+    [Parameter]
+    public bool Unstyled { get; set; }
```

Comment:
Move this parameter up to the BaseLinkComponent class.

Also mention it in the API section on the Link page in the documentation.